docker/sequencer.Dockerfile

```diff
@@ -9,4 +9,6 @@ RUN apt-get update \
 COPY target/$TARGETARCH/release/sequencer /bin/sequencer
 RUN chmod +x /bin/sequencer
 
-CMD [ "/bin/sequencer"]
+CMD ["/bin/sequencer", "--", "http"]
+HEALTHCHECK CMD curl --fail http://localhost:$ESPRESSO_SEQUENCER_API_PORT/healthcheck  || exit 1
```

Comment:
Does it need to be specified in both the Dockerfile and the docker compose file?

Reply:
I don't think so. I can remove it from the compose file

Reply:
Good idea to add them to the dockerfiles instead, I think.